crawler_mcp/core/connection_pool.py

```
@@ -0,0 +1,320 @@
+
+    async def close(self) -> None:
+        """Close all connections in the pool."""
+        logger.info("Closing connection pool")
+
+        # Cancel health check task
+        if self._health_check_task:
+            self._health_check_task.cancel()
+            try:
+                await self._health_check_task
+            except asyncio.CancelledError:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(asyncio.CancelledError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(asyncio.CancelledError)`

(SIM105)